modules/enet/networked_multiplayer_enet.cpp

```diff
@@ -248,7 +248,7 @@ void NetworkedMultiplayerENet::poll() {
 				int *new_id = memnew(int);
 				*new_id = event.data;
 
-				if (*new_id == 0) { // Data zero is sent by server (enet won't let you configure this). Server is always 1.
+				if (*new_id == 0) { // Data zero is sent by server (Enet won't let you configure this). Server is always 1.
```

Comment:
If changing this, then it should be done all the way. It's capitalized "ENet": http://enet.bespin.org/